src/develop/imageop_math.h

```diff
@@ -175,6 +175,22 @@ static inline int FC(const size_t row, const size_t col, const uint32_t filters)
 #define BLUE 2
 #define ALPHA 3
 
+static inline float get_processed_maximum(dt_dev_pixelpipe_iop_t *piece)
```

Comment:
naming conventions?

Reply:
not sure about this. Leading underscore? Or dt_iop_xxx?

Reply:
I don't know. `dt_pixelpipe_` (and move it to pixel pipe.h)? `dt_dev_pixelpipe_` is so looong. We don't have full consistency anyway, but I'd say for new additions we should at least have `dt_` to avoid confusion (for nonlocal, like this). Do you agree?

Reply:
Chose `dt_iop_` as "prefixes"